_data/simple-icons.json

```diff
@@ -9487,6 +9487,12 @@
             "source": "https://opnsense.org/about/legal-notices/",
             "guidelines": "https://opnsense.org/about/legal-notices/"
         },
+        {
+            "title": "Oppo",
```

Comment:
```suggestion
            "title": "OPPO",
```

See this: https://www.oppo.com/en/about/.

They call themself `OPPO`.

Reply:
Fixed